api/MimeTypeArray.json

```diff
@@ -157,14 +151,8 @@
               "version_added": "4"
             },
             "safari_ios": "mirror",
-            "samsunginternet_android": {
-              "version_added": "1.0",
-              "notes": "Before Samsung Internet 7.0, method parameters were optional"
-            },
-            "webview_android": {
-              "version_added": "≤37",
-              "notes": "Before version 59, method parameters were optional"
-            },
+            "samsunginternet_android": "mirror",
```

Comment:
It looks like there are still about 15 Samsung Internet changes (see [here](https://github.com/mdn/browser-compat-data/actions/runs/14399902190/job/40383155630?pr=26471#step:7:1), search for `samsunginternet_android`).

Can we revert these and land separately?